torch/distributed/fsdp/_init_utils.py

```diff
@@ -506,6 +506,30 @@ def _init_prefetching_state(
     return state
 
 
+@no_type_check
+def _init_extension(state: _FSDPState, device_mesh: DeviceMesh = None) -> _FSDPState:
+    # TODO: we need to add additional check once we support FSDP + PiPPy.
+    # This check is currently sufficient, since we only support FSDP + TP.
+    if device_mesh:
+        state._enable_extension = (
+            mesh_resources.get_parent_mesh(state._device_mesh) is not None
+        )
+
+        if state._enable_extension:
+            try:
```

Comment:
We don't need this try/except for the import, right? There is no way `_init_extension` exist but those extensions are not in the code base.